serverlessworkflow/modules/ROOT/pages/core/defining-an-input-schema-for-workflows.adoc

```diff
@@ -1 +1,58 @@
-//= Defining an input schema for workflows
\ No newline at end of file
+Defining an input schema for workflow
+=====================================
+:compat-mode!:
+
+DataInputSchema is one of the parameters in the Workflow definition as per the Serverless Workflow specification.
+
+The dataInputSchema property can be used to validate the workflow data input against a defined JSON Schema. dataInputSchema can be defined as follows:
+[source,json]
+----
+"dataInputSchema": {
+   "schema": "URL_to_json_schema",
+   "failOnValidationErrors": false
+}
+----
+
+"schema" property is an URI, which holds the path to the JSON schema used to validate the workflow data input. The JSON schema file should be placed in the resources section of the project.
+
+An example for a workflow definition with datainputschema can be found here- https://github.com/kiegroup/kogito-runtimes/blob/main/quarkus/extensions/kogito-quarkus-serverless-workflow-extension/kogito-quarkus-serverless-workflow-integration-test/src/main/resources/expression.sw.json#L16[GitHub repository]
+
+You can find a sample JSON schema here- https://github.com/kiegroup/kogito-runtimes/blob/main/quarkus/extensions/kogito-quarkus-serverless-workflow-extension/kogito-quarkus-serverless-workflow-integration-test/src/main/resources/schema/expression.json[GitHub repository]
+
+When a workflow definition has a dataInputSchema attribute defined,the application should generate an OpenAPI file (http://localhost/q/swagger-ui) referencing this schema file, which helps to correctly define the input data for the workflows.
```

Comment:
In this case, the application is the Kogito SW, so we will generate the OpenAPI file if the user has the [Quarkus dependency in their project](https://quarkus.io/guides/openapi-swaggerui#expose-openapi-specifications). That's what we want to highlight here.

Reply:
@ricardozanini Thank you Zanini, will make the changes accordingly.